crates/tabby-common/src/index/mod.rs

```diff
@@ -178,6 +178,21 @@ impl IndexSchema {
         ])
     }
 
+    /// Build a query to find the document with the given `doc_id`, include chunks.
```

Comment:
why move order?

Reply:
The `doc_query_with_chunks` function should remain adjacent to the `doc_query` function and not be separated by `doc_indexed_after` and other functions.